invenio_administration/assets/semantic-ui/translations/invenio_administration/i18next-scanner.config.js

```diff
@@ -0,0 +1,63 @@
+// This file is part of React-Invenio-Deposit
+//
+// Invenio-administration is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+// list of func used to
+// mark the strings for translation
+const { languages } = require("./package.json").config;
+
+const funcList = ["i18next.t"];
+const extensions = [".js", ".jsx"];
+
+module.exports = {
```

Comment:
We are copy/pasting this code and files below everywhere... @zzacharo @kpsherva did we chat about making it re-usable?

Reply:
yes we did, there is even a ticket. I think its the time!